### DISCUSSION TITLE
Default username/password

### DISCUSSION THREAD

**Discussion by AndyIsHereBoi** [tteck/Proxmox#461] [Help Needed] [answered]
Can I have the default username or password for the Nginx Proxy Manager and Vaultwarden installs?

**Comment by AndyIsHereBoi** [comment, marked as answer]
Nevermind. I just realised that it already logs you in

**Reply by nacho-domatix** [reply]
Not sure what is the answer to this. What is actually the default user/pass?

**Reply by tteck** [reply]
He was using autologin, therefore there's no password
